printer_mods/README.md

```diff
@@ -123,6 +123,8 @@ like so:
 | | [Voron0_MGN9C_X_Axis](./hartk1213/Voron0_MGN9C_X_Axis) | This mod, replaces the stock MGN7H for MGN9C on the V0  | :heavy_check_mark:| :x: | :x: |:x:|
 | | [Voron2.4_Pins_Mod](./hartk1213/Voron2.4_Pins_Mod) | This mod replaces all the M5 screws for all idlers/pulleys/bearings with 5mm pins  | :x:| :x: | :heavy_check_mark: |:x:|
 | | [Voron2.4_Y_Endstop_Relocation](./hartk1213/Voron2.4_Y_Endstop_Relocation) | This mod relocates the Y endstop to the top A drive piece, to work with Arkeets MGN12 mod  | :x:| :x: | :heavy_check_mark: |:x:|
+| | [Voron2.4_GE5C](./hartk1213/Voron2.4_GE5C) | This mod replaces the stack of printed z joints with a metal spherical bearing  | :x:| :x: | :heavy_check_mark: |:x:|
```

Comment:
Inside the square brackets (what is actually shown as the title to the user), please use spaces instead of underscores, so it doesn't make the table column too wide.  Instead of

    [Voron2.4_GE5C](./hartk1213/Voron2.4_GE5C)

Do

    [Voron2.4 GE5C](./hartk1213/Voron2.4_GE5C)

Also please add an `:x:` for the last (Trident) column.

Same thing for the next line.

Reply:
fixed and removed any other underscores as well from my other stuff